hlf/chaincode/admission.md

```diff
@@ -0,0 +1,171 @@
+# Hyperledger Admission Api
+
+In the following, the chaincode api for the "Admission"-Chaincode is described.
+This contains all Transactions and Models provided for this domain.
+The Errors returned are defined [here](errors.md#Errors).
+
+## ContractName "UC4.Admission"
+
+## Transactions
+
+### Add Admission
+- ID = addAdmission
+- Send
+    - jsonAdmission :: [Admission](#Admission)
+- Receive
+    - [Admission](#Admission)
+      -  Description: Done, returns the submitted data, decorated with an admissionId.
+
+    - [DetailedError](errors.md#DetailedError) 
+      ```json
+      {
+        "type": "HLUnprocessableEntity",
+        "title": "The following parameters do not conform to the specified format",
+        "invalidParams": [
+          {
+            "name": "string",
+            "reason": "string"
+          }
+        ]
+      }
+      ```
+       - Description: This error is returned, if the given parameters could not be parsed. If some attributes within the json are not well formatted, they are listed in invalidParams.  
+       For detailed informations see [parameter checks](#parameterChecks).
+
+    - [DetailedError](errors.md#DetailedError) 
+      ```json
+      {
+        "type": "HLInvalidEntity",
+        "title": "The following parameters produced some semantic error",
+        "invalidParams": [
+          {
+            "name": "string",
+            "reason": "string"
+          }
+        ]
+      }
+      ```
+      - Description: This error is returned, if the given parameters produce some error.  
+      For detailed informations see [semantic checks](#semanticChecks).

```

Comment:
```suggestion
```
We decided that this would be captured by HLUnprocessableEntity, right?

Reply:
Lagom agrees

Reply:
Do we still change the title?

Reply:
type changed in https://github.com/upb-uc4/api/pull/83/commits/98fd8d1f615d3aab3f2868d30f18ac2295dda6ed

Reply:
completely removed extra error